src/ui/public/error_allow_explicit_index/error_allow_explicit_index.html

```diff
@@ -1,17 +1,21 @@
 <div class="app-container error-multi-allow-explicit-index">
-  <h3>
+  <h3 class="kuiVerticalRhythm">
```

Comment:
Can you remove these the old classes? I've got a PR for styling these errors.

Reply:
👍 thanks!